CONTRIBUTING.md

```diff
@@ -134,26 +141,69 @@ For additional information on using the `dep` tool, see
 
 ## Patch format
 
+### General format
+
 Beside the `Signed-off-by` footer, we expect each patch to comply with the
 following format:
 
 ```
-Subsystem: One line change summary
+subsystem: One line change summary
 
-More detailed explanation of your changes: Why and how.
+More detailed explanation of your changes (why and how)
+that spans as many lines as required.
 
-See:
-    http://chris.beams.io/posts/git-commit/
+A "Fixes #XXX" comment listing the GitHub issue this change resolves.
+This comment is required for the main patch in a sequence. See the following examples.
 
-For some more good advice, and the Linux Kernel document:
-    https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/SubmittingPatches
+Signed-off-by: <contributor@foo.com>
+```
 
-Fixes: #nnn
+The body of the message is not a continuation of the subject line and is not used to extend the subject line beyond its character limit. The subject line is a complete sentence and the body is a complete, standalone paragraph.
+
+### Subsystem
+
+The "subsystem" describes the area of the code that the change applies to. It does not have to match a particular directory name in the source tree because it is a "hint" to the reader. The subsystem is generally a single word. Although the subsystem must be specified, it is not validated. The author decides what is a relevant subsystem for each patch.
+
+Examples:
+
+| Subsystem | Description |
+|--|--|
+| `build` | `Makefile` or configuration script change |
+| `cli` | Change affecting command line options or commands |
+| `docs` | Documentation change |
+| `logging` | Logging change |
+| `vendor` | [Re-vendoring](#re-vendor-prs) change |
+
+To see the subsystem values chosen for existing commits:
 
-Signed-off-by: <contributor@foo.com>
 ```
+$ git log --no-merges --pretty="%s" | cut -d: -f1 | sort -u
+```
+
+### Advice
+
+It is recommended that each of your patches fixes one thing. Smaller patches
+are easier to review, more likely accepted and merged, and problems are more
+likely to be identified during review.
+
+A PR can contain multiple patches. These patches should generally be related to the [main patch](#main-patch) and the overall goal of the PR. However, it is also acceptable to include additional or [supplementary patches](#supplementary-patch) for things such as:
+
+- Formatting (or whitespace) fixes.
```

Comment:
I suggest removing the periods after each list item on lines 191-194.

Reply:
Thanks @klynnrif - branch updated.